Sources/SPMTestSupport/TestWorkspace.swift

```diff
@@ -288,6 +296,42 @@ public final class TestWorkspace {
         result(graph, diagnostics)
     }
 
+    public func checkIsResolutionRequired(
+        pins: [PackageReference: CheckoutState],
+        managedDependencies: [ManagedDependency],
+        _ check: (Bool, DiagnosticsEngine) -> ()
```

Comment:
maybe add labels to the types for clarity ?

Reply:
Done.